README.md

```diff
@@ -889,8 +895,55 @@ Similarly, when running tests in PyCharm, its own proprietary test discovery
 process may also increase the chance of the `AzulTestCase` context manager
 causing a failure.
 
-If these failures occur, add the warning to the list of permitted warnings found
-in [`AzulTestCase`](test/azul_test_case.py) and commit the modifications. 
+If these failures occur, add the warning to the list of permitted warnings
+found in [`AzulTestCase`](test/azul_test_case.py) and commit the modifications. 
+
+
+## Installing Python 3.8.3 on macOS BigSur
+
+[pyenv Big Sur GitHub issue](https://github.com/pyenv/pyenv/issues/1740)
+
+Users of macOS 11 or later may encounter a `build failed` error when installing
+Python through pyenv. A patch was made available to remedy this:
+
+First, ensure that bzip2 and any other requirements for the Python build
+environment are met.
+See [pyenv wiki](https://github.com/pyenv/pyenv/wiki#suggested-build-environment)
+for details:
+
+```
+brew install openssl readline sqlite3 xz zlib bzip2
+```
+
+Follow any additional steps that `brew` prompts for at the end of the
+installation. These should include modifying path variables `LDFLAGS`
+and `CPPFLAGS`. The commands from the `brew` output to modify the
+aforementioned path variables can be placed in `~/.bash_profile` to make the
+change persistent.
```

Comment:
Wrapping is still wrong. For example:

```suggestion
Follow any additional steps that `brew` prompts for at the end of the
installation. These should include modifying path variables `LDFLAGS` and
`CPPFLAGS`. The commands from the `brew` output to modify the aforementioned
path variables can be placed in `~/.bash_profile` to make the change
persistent.
```

Reply:
I believe wrapping should be fixed with the latest edit.
I don't know if it was right but I set the line length setting in the Pycharm Preferences to 80.
It seemed right since it matched up to the max length of most other lines.
I thought it was max length 79 but I re-read the Contributing guidelines and realized that I misinterpreted it.

Reply:
The version I commented on was wrong under both setting, 79 or 80, so that alone doesn't explain the mistake.